### PR TITLE
Updates #422 - Modifies progresscard based on modules completed

### PR DESCRIPTION
**Pull request by pmulard**
Also adds test coverage for progresscard.

### CODE REVIEW

**Review by ggwadera — APPROVED** [review]
Nice work 👍

File: components/ProgressCard.tsx [hunk 1]
```diff
@@ -57,14 +57,28 @@ const ProgressSVG: React.FC<Props> = ({ progressCount }) => {
 }
 
 const ProgressCard: React.FC<Props> = ({ progressCount }) => {
+  const setHeaderFromProgress = () => {
+    if (progressCount <= 13) {
+      // Has not finished 1st module yet
+      return `You're off to a great start!`
+    } else if (progressCount <= 50) {
```
Comment (songz) [inline components/ProgressCard.tsx:64]: FYI, if you are returning, `else` is redundant.

**Review by songz — APPROVED** [review]

**Review by hwong0305 — APPROVED** [review]

File: components/ProgressCard.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import ProgressCard from './ProgressCard'
+
+describe('ProgressCard component', () => {
+  it('should render correct lesson progress for 0% completion', () => {
+    const { container } = render(<ProgressCard progressCount={0} />)
+    expect(container).toMatchSnapshot()
```
Comment (Ulisseus) [inline components/ProgressCard.test.js:8]: No need to check against new snapshot every time. Testing library has more advanced [queries](https://testing-library.com/docs/queries/about/). For example, you can check if `You're off to a great start!` is present on the screen:

` expect(screen.getByText("You're off to a great start!")).toBeTruthy()`

Or if your component changes its state over time. 
` await waitFor(() => screen.getByText('You're off to a great start!'))`

It's just for your information. Your PR is fine as it is.

**Review by Ulisseus — APPROVED** [review]

File: components/ProgressCard.test.js (hunk 2 above)
Comment (pmulard) [reply components/ProgressCard.test.js:8]: Thank you for explaining this. Still very fresh using Jest.

File: components/ProgressCard.tsx (hunk 1 above)
Comment (pmulard) [reply components/ProgressCard.tsx:64]: Yes, good catch.

**Pull request merged**